airbyte-webapp/src/components/ui/ListBox/ListBox.tsx

```diff
@@ -15,6 +17,7 @@ const DefaultControlButton = <T,>({ selectedOption }: ListBoxControlButtonProps<
 export interface Option<T> {
   label: string;
   value: T;
+  icon?: IconProp;
```

Comment:
What type of icons would be passed in?

Currently only Font Awesome icons work, but I'm guessing that would be extended to SVG icons. Would like more clarification!